### ISSUE TITLE
NNID Software Question

### ISSUE THREAD

**Issue by ZalexZTK** [Plailect/Guide#560]
I didn't know where to put this and get reliable information, so sorry if this isn't the place to ask this.

I want to downgrade my O3Ds from 11.0 to 10.4, but not if I lose all my software on the Ds I'm trying to downgrade. I have a lot of games on there tied to my current NNID, so if I delete my NNID from the system, then re-enter it into my hacked Ds, will I be able to download the games, then just system transfer back into my Stock Ds?

Basically, is there any way to transfer the games via NNID. Are the games saved to that NNID, so that I can download them on different devices? Or do I call Nintendo or something? Really don't want to lose my games, but don't really want to hardmod either. Thanks!

**Comment by pbanj** [comment]
you are not downgrading from 11.0 to 10.4. when you do the dsiware downgrade it only downgrades the native firm. after that it allows you to downgrade to 9.2, and then to 2.1. downgrading itself does not format/make you lose your games. if you are worried about saves and what not you can back them up with jksm.

when you do the transfer everything that is on the source(hacked) system will go to the target(not hacked) system. everything that is on the target from before the transfer will be wiped, the nnid that was on it will still be linked but just logged out. after you hack the target system you could format it and then log back into the original nnid and redownload all of your stuff.

you can also do what some people do. transfer from the non hacked to the hacked, wait a week, buy fieldrunners, set up fieldrunners and say steelhax, then transfer back to the non hacked so they can then hack it. it leaves the target system with all of its stuff and allows it to be hacked.

**Comment by ZalexZTK** [comment]
Oh okay. Thank you very much for the info! Now I can confidently attempt the downgrade!
